### DISCUSSION TITLE
pop up window "Would you like to use a compile_commands.json file to auto-configure IntelliSense for this folder"

### DISCUSSION THREAD

**Discussion by congmingyige** [microsoft/vscode-cpptools#8942] [Q&A] [answered]
When I open VSCode, when folder has opened, it always pops up a window, the content is shown below:
> English: Would you like to use a compile_commands.json file to auto-configure IntelliSense for this folder?
Chinese: 是否要使用 compile_commands.json 文件 为此文件夹自动配置 IntelliSense?

How should I do to avoid showing the window?

**Comment by sean-mcmanus** [comment, marked as answer]
Choose the Yes or No option (I assume No) -- does that work for you?

**Reply by congmingyige** [reply]
What I write previously is incomplete. The thing that the window opens all the time when opening a folder(project) troubles me, it is not the window content that troubles me. It seems that after choosing no(or others), the window will not open again. But how to avoid opening this window when opening other folders(projects) is a problem.
What's more, may I ask what is the difference between choosing yes and no?

**Reply by sean-mcmanus** [reply]
Choosing "Yes" will configure IntelliSense using compileCommands and choosing "No" won't.

There is no way currently to disable the prompt. You could file an issue to request a setting to disable it.

**Reply by congmingyige** [reply]
> Choosing "Yes" will configure IntelliSense using compileCommands and choosing "No" won't.
> 
> There is no way currently to disable the prompt. You could file an issue to request a setting to disable it.

Thanks!
